restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/http/AccountsEndpointTest.java

```diff
@@ -568,6 +583,79 @@ public void testPasswordStrengthForAccountUpdate() {
         assertEquals(200, response.getStatus());
     }
 
+    @Test
+    public void testCreateAccountInSpecificOrganization() {
+    	// child should not be able to create account in specified org that it does not belong to
```

Comment:
this is just a general comment: not sure why you group all this scenarios in one testcase (i think they sequence you are testing now is not related, just in theme), and put a comment. It may be different testcases with proper test intention as part of test case emthod name.

Reply:
intention is to test both 
1. permission
2. creation process

reason i would avoid a separate test case is performance, it will take time to gear up a new archive just to separately test above 2 scenarios

Reply:
not sure performance is a reason, since i was thinking in creating new test methods/cases inside same test class. For a single test class, there is single container, and deployment of restcomm...but again, this is general comment on testing practices

Reply:
right @jaimecasero agreed on practices, i have split them into separate cases